lib/Client.php

```diff
@@ -90,6 +90,31 @@ public function __construct(
 		}
 	}
 
+	public function requestUserInfo($attribute = null) {
+		if (true) {
+			$resp = $this->requestTokenExchange(
+				$this->getRefreshToken(),
+				'urn:ietf:params:oauth:token-type:refresh_token',
```

Comment:
should be access token to have a solution which will work with our clients

Reply:
```
(
    [error_description] => Invalid request: Invalid token type for token exchange, must be a RefreshToken
    [error] => invalid_request
)
```